rewrite.yml

```diff
@@ -16,6 +16,8 @@
 type: specs.openrewrite.org/v1beta/recipe
 name: org.apache.druid.RewriteRules
 recipeList:
+  - org.openrewrite.apache.commons.math.UpgradeApacheCommonsMath_2_3
+  - org.openrewrite.apache.commons.collections.UpgradeApacheCommonsCollections_3_4
```

Comment:
I don't see common-collections 3 being used at all - but it was pulled in into the project:
* opencsv 5.8 ->  commons-beanutils:1.9.4 -> commons-collections:3.2.2
* kafka_test -> commons-validator:commons-validator:jar:1.7:test -> commons-collections:3.2.2

it seems to me that there are only 2 classes being used from commons-collection4